docs/tls/README.md

```diff
@@ -117,13 +139,14 @@ encryption_key
 Generate public certificate
 
 ```
-certtool.exe --generate-self-signed --load-privkey private.key --template cert.cnf --outfile public.crt 
+certtool.exe --generate-self-signed --load-privkey private.key --template cert.cnf --outfile public.crt
 ```
 
 ## 4. Install third-party CAs
 
 Minio can be configured to connect to other servers, whether Minio nodes or servers like NATs, Redis. If these servers use certificates that are not registered in one of the known certificates authorities, you can make Minio server trust these CAs by dropping these certificates under Minio config path (`~/.minio/certs/CAs/` on Linux or `C:\Users\<Username>\.minio\certs\CAs` on Windows).
 
 # Explore Further
-* [Minio Quickstart Guide](https://docs.minio.io/docs/minio-quickstart-guide)
 * [Minio Client Complete Guide](https://docs.minio.io/docs/minio-client-complete-guide)
+* [Generate Let's Encrypt Certificate](https://docs.minio.io/docs/generate-let-s-encypt-certificate-using-concert-for-minio)
```

Comment:
concert is deprecated we should remove, we should point to certbot.

Reply:
The link actually points to certbot doc, not sure why link says concert

Reply:
perhaps we should fix the slug name.